keras_retinanet/bin/convert_model.py

```diff
@@ -74,7 +78,9 @@ def main(args=None):
     models.check_training_model(model)
 
     # convert the model
-    model = models.convert_model(model, nms=args.nms, class_specific_filter=args.class_specific_filter, anchor_params=anchor_parameters)
+    model = models.convert_model(model, nms=args.nms, class_specific_filter=args.class_specific_filter, anchor_params=anchor_parameters,
```

Comment:
Also nitpicky, but our style for long lines is:
```python
foo(
    bar,
    baz=1,
    foobar=2
)
```
It's more lines, but IMO more readable.

Reply:
Done

Reply:
Not exactly what I had in mind, could you have another look at this?

Reply:
Ok. I did it similarily to your other function calls. Like this ?